FogCreek/Glitch-Community
# tasty-quail: support local apis

## Links
* Remix link: https://tasty-quail.glitch.me/

## Changes:
* Working to clear up what `RUNNING_ON` (and `RUNNING_LOCALLY`) so that local API servers can be used for remixes or local development

## How To Test:
* Setting `RUNNING_ON=local` should use an environment with the API URL set to `tasty-quail.glitch.fwd.wf`

## Feedback I'm looking for:

## Things left to do before deploying:
